.github/workflows/pr.yaml

```diff
@@ -53,7 +53,7 @@ jobs:
         pip3 install -r requirements.txt
         apiserver=`kubectl config view --minify -o jsonpath='{.clusters[0].cluster.server}'`
         echo "API_SERVER_URL:$apiserver"
-        python3 provider-kubeconfig.py -s $apiserver create $KUBEPLUS_NS
+        python3 provider-kubeconfig.py create -s $apiserver -n $KUBEPLUS_NS
```

Comment:
About putting Namespace as a parameter under a flag - the problem is it will make Namespace an optional parameter. We want to keep Namespace a required parameter. Keeping it a positional parameter achieves this. So I suggest we keep Namespace as a positional parameter.

Reply:
Updated to keep namespace as a positional parameter.